blob/azureblob/azureblob.go

```diff
@@ -216,6 +216,8 @@ func (o *ServiceURLOptions) withOverrides(urlValues url.Values) (*ServiceURLOpti
 				return nil, err
 			}
 			retval.IsLocalEmulator = isLocalEmulator
+		case "storage_account":
```

Comment:
So, this will override the AccountName for the ServiceURLOptions, and thus in the ServiceURL, but not in the credentials; see newCredInfoFromEnv below.

Does overriding this only make sense if you're using an authentication mechanism that doesn't include AccountName (e.g., not shared key)?